change/@ni-nimble-angular-148cc61e-756b-4f60-8409-3c67783094a7.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I don't see any angular changes, did you forget to push them?

Reply:
[Mert recommended](https://github.com/ni/nimble/pull/1355#discussion_r1265879791) in a previous review that changes to the angular testing wasn't necessary since the implementation of the new states didn't change the point of the tests. I forgot to delete that change file after making the fix Mert recommended, I'll do that in my next commit.